index.js

```diff
@@ -42,7 +42,7 @@ var metrics = require('metrics');
 var packageJSON = require('./package.json');
 
 var Gossip = require('./lib/gossip');
-var Suspicion = require('./lib/gossip/suspicion');
+var StateTransitions = require('./lib/gossip/state_transitions');
```

Comment:
Not really blocking because it is used both in this project, but normally nodejs modules name files with `-` in favor of `_`. In ringpop they are both used which is already quite messy.